virtualenv_embedded/activate.sh

```diff
@@ -54,11 +60,12 @@ if ! [ -z "${PYTHONHOME+_}" ] ; then
 fi
 
 if [ -z "${VIRTUAL_ENV_DISABLE_PROMPT-}" ] ; then
-    _OLD_VIRTUAL_PS1="$PS1"
+    _OLD_VIRTUAL_PS1="${PS1:-}"
```

Comment:
I think it's better to go with `${PS1-}` we just want to guard against unset variables. Why add the side-effect of setting PS1?

And look: this construct is already employed in line 62 above --this should remove any concerns about portability

Reply:
@jakub-bochenski Out of curiosity, is there a difference between `${FOO-}` and `${FOO:-}`? I used the `:-` for years and that was the first time I encountered the simple minus.

Reply:
Haste makes waste -- I was wrong :)

The difference between `:-` and `-` is in treatment of null values, none of those does an assignment (you need either `=` or `:=`). Reference: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Still I think it might be better to use `-` because
 - it's already used
 - it only affects unset variables and leaves null as null, so it's a tighter fit